bucket/maven.json

```diff
@@ -17,7 +17,7 @@
         "re": "<b>([\\d.]+)</b>"
     },
     "autoupdate": {
-        "url": "https://archive.apache.org/dist/maven/maven-$majorVersion/$version/binaries/apache-maven-$version-bin.zip",
+        "url": "https://www.apache.org/dist/maven/maven-$majorVersion/$version/binaries/apache-maven-$version-bin.zip",
         "extract_dir": "apache-maven-$version",
         "hash": {
             "url": "$url.sha1"
```

Comment:
```suggestion
            "url": "$url.sha512"
```

Reply:
thanks, it's my negligence.

the suggested change already merge, pull again.